item-spec/json-schema/item.json

```diff
@@ -67,6 +67,48 @@
                   "description": "The searchable date/time of the assets, in UTC (Formatted in RFC 3339) ",
                   "type": "string",
                   "format": "date-time"
+                },
+                "title": {
+                  "title": "Item Title",
+                  "description": "A human-readable title describing the item.",
+                  "type": "string"
+                },
+                "license": {
+                  "title": "Item Licenses",
+                  "type": "string"
+                },
+                "providers": {
```

Comment:
Would it make sense to import these definitions from a central place (e.g. collection JSON schema?)

Reply:
Yes... I'll see what I can do about this outside this PR.